From: Theodore Ts'o <tytso@mit.edu>
Date: Wed, 22 Aug 2018 00:52:08 -0400
Subject: Make sure all release note files have a single trailing empty line

This is needed so we can assemble a complete release notes file via a
command like:

cat $(/bin/ls -1 doc/RelNotes/v*.txt | tac) | gzip -9n > NEWS.gz

Signed-off-by: Theodore Ts'o <tytso@mit.edu>
Origin: Upstream, https://git.kernel.org/pub/scm/fs/ext2/e2fsprogs.git/commit/?h=maint&id=b1d874a6b851
---
 doc/RelNotes/v1.02.txt   | 1 +
 doc/RelNotes/v1.34.txt   | 1 +
 doc/RelNotes/v1.35.txt   | 1 +
 doc/RelNotes/v1.36.txt   | 1 +
 doc/RelNotes/v1.38.txt   | 1 +
 doc/RelNotes/v1.40.txt   | 1 +
 doc/RelNotes/v1.42.txt   | 1 +
 doc/RelNotes/v1.43.6.txt | 1 -
 doc/RelNotes/v1.43.7.txt | 1 -
 doc/RelNotes/v1.43.8.txt | 1 -
 doc/RelNotes/v1.43.9.txt | 1 -
 doc/RelNotes/v1.44.0.txt | 1 +
 doc/RelNotes/v1.44.1.txt | 1 +
 doc/RelNotes/v1.44.2.txt | 1 +
 doc/RelNotes/v1.44.3.txt | 1 +
 doc/RelNotes/v1.44.4.txt | 1 +
 16 files changed, 12 insertions(+), 4 deletions(-)

diff --git a/doc/RelNotes/v1.02.txt b/doc/RelNotes/v1.02.txt
index 83fefdc..27dce52 100644
--- a/doc/RelNotes/v1.02.txt
+++ b/doc/RelNotes/v1.02.txt
@@ -22,3 +22,4 @@ Patch lsattr so that it won't hang when checking a named pipe.
 Minor compilation fixes:
 	* Fix the order of libraries that were linked in debugfs.
 	* Allow the sources to be compiled with -ansi turned on.
+
diff --git a/doc/RelNotes/v1.34.txt b/doc/RelNotes/v1.34.txt
index 77db125..f61095c 100644
--- a/doc/RelNotes/v1.34.txt
+++ b/doc/RelNotes/v1.34.txt
@@ -127,3 +127,4 @@ variables are set, which specifies which I/O operations are logged and
 a block number to watch, respectively.  The log messages are sent to
 stderr by default, unless a filename is specified via the
 TEST_IO_LOGFILE environment variable.
+
diff --git a/doc/RelNotes/v1.35.txt b/doc/RelNotes/v1.35.txt
index ea56f74..2a4c520 100644
--- a/doc/RelNotes/v1.35.txt
+++ b/doc/RelNotes/v1.35.txt
@@ -163,3 +163,4 @@ Fixed various compiler warnings.
 
 Add portability fixes for FreeBSD and for using fsctl under Darwin to
 support ext2 ioctl's.
+
diff --git a/doc/RelNotes/v1.36.txt b/doc/RelNotes/v1.36.txt
index 8268d41..912bad2 100644
--- a/doc/RelNotes/v1.36.txt
+++ b/doc/RelNotes/v1.36.txt
@@ -299,3 +299,4 @@ Drop the sparc assembly bitwise operations; it's less efficient
 than the GCC 3.4 compile code and triggers compiler warnings on
 sparc64.  Thanks to Matthias Andree for his analysis and suggestions.
 (Addresses Debian Bug #232326)
+
diff --git a/doc/RelNotes/v1.38.txt b/doc/RelNotes/v1.38.txt
index 92eb342..d8ab030 100644
--- a/doc/RelNotes/v1.38.txt
+++ b/doc/RelNotes/v1.38.txt
@@ -135,3 +135,4 @@ header files have not be installed in the system include directories.
 (Addresses Sourceforge Bug: #1180572)
 
 Fixed gcc -Wall nits.
+
diff --git a/doc/RelNotes/v1.40.txt b/doc/RelNotes/v1.40.txt
index 4954052..08608d2 100644
--- a/doc/RelNotes/v1.40.txt
+++ b/doc/RelNotes/v1.40.txt
@@ -1040,3 +1040,4 @@ which require the backups to be touched.
 
 Add new function to libext2fs, ext2fs_default_journal_size(), which
 returns the default journal size.
+
diff --git a/doc/RelNotes/v1.42.txt b/doc/RelNotes/v1.42.txt
index ae5c930..f0d2dd2 100644
--- a/doc/RelNotes/v1.42.txt
+++ b/doc/RelNotes/v1.42.txt
@@ -1687,3 +1687,4 @@ Allow ext2fs_get_memalign() to compile on systems that don't have
 posix_memalign().
 
 Fixed a namespace leak in libext2fs (tdb_null).
+
diff --git a/doc/RelNotes/v1.43.6.txt b/doc/RelNotes/v1.43.6.txt
index 8bfe25e..1050d5e 100644
--- a/doc/RelNotes/v1.43.6.txt
+++ b/doc/RelNotes/v1.43.6.txt
@@ -68,4 +68,3 @@ artifacts.
 
 Removed legacy entries for ext4dev in the default mke2fs.conf file.
 
-
diff --git a/doc/RelNotes/v1.43.7.txt b/doc/RelNotes/v1.43.7.txt
index 385de5a..55e434f 100644
--- a/doc/RelNotes/v1.43.7.txt
+++ b/doc/RelNotes/v1.43.7.txt
@@ -27,4 +27,3 @@ Test descritions have been shorted to fit in 80 columns.
 
 Fix various tests to avoid leaving temp files in /tmp.
 
-
diff --git a/doc/RelNotes/v1.43.8.txt b/doc/RelNotes/v1.43.8.txt
index 9edd69b..cb26a16 100644
--- a/doc/RelNotes/v1.43.8.txt
+++ b/doc/RelNotes/v1.43.8.txt
@@ -46,4 +46,3 @@ Clean up Makefile output.
 Fix compatibility problems with glibc 2.27 casued by a collision with
 copy_file_range().
 
-
diff --git a/doc/RelNotes/v1.43.9.txt b/doc/RelNotes/v1.43.9.txt
index d26003f..6dbe0db 100644
--- a/doc/RelNotes/v1.43.9.txt
+++ b/doc/RelNotes/v1.43.9.txt
@@ -13,4 +13,3 @@ Fix a build failure in lib/ext2fs/swapfs.c on big-endian systems.
 
 Fix various Debian packaging issues.  (Addresses Debian Bug #269569).
 
-
diff --git a/doc/RelNotes/v1.44.0.txt b/doc/RelNotes/v1.44.0.txt
index 5fa5299..59cd802 100644
--- a/doc/RelNotes/v1.44.0.txt
+++ b/doc/RelNotes/v1.44.0.txt
@@ -79,3 +79,4 @@ E2fsck is now much faster for bigalloc file systems when scanning
 extents for bigalloc file systems.  (Addresses Google Bug #36886699)
 
 Update Czech, Spanish, French, Malay, and Ukrainian translations.
+
diff --git a/doc/RelNotes/v1.44.1.txt b/doc/RelNotes/v1.44.1.txt
index 4716974..4503c92 100644
--- a/doc/RelNotes/v1.44.1.txt
+++ b/doc/RelNotes/v1.44.1.txt
@@ -45,3 +45,4 @@ Fix some test failures on GNU Hurd caused by a different default block
 size.
 
 Add new regression tests.
+
diff --git a/doc/RelNotes/v1.44.2.txt b/doc/RelNotes/v1.44.2.txt
index 1dca63c..7b5c774 100644
--- a/doc/RelNotes/v1.44.2.txt
+++ b/doc/RelNotes/v1.44.2.txt
@@ -50,3 +50,4 @@ Fix the f_bigalloc_badinode and f_bigalloc_orphan_list regression tests
 so they uses mke2fs and ebugfs from the build tree.  They may not exist
 in the users PATH, and if they not or if they are extremely
 old/obsolete, those tests would fail.
+
diff --git a/doc/RelNotes/v1.44.3.txt b/doc/RelNotes/v1.44.3.txt
index 7312621..e68032c 100644
--- a/doc/RelNotes/v1.44.3.txt
+++ b/doc/RelNotes/v1.44.3.txt
@@ -103,3 +103,4 @@ Fixed Coverity, sparse, gcc -Wall, and clang warnings/nits.
 
 Update Czech, Dutch, Spanish, French, Polish, Swedish, Ukrainian, and
 Vietnamese translations.
+
diff --git a/doc/RelNotes/v1.44.4.txt b/doc/RelNotes/v1.44.4.txt
index dbe6e5d..db130c6 100644
--- a/doc/RelNotes/v1.44.4.txt
+++ b/doc/RelNotes/v1.44.4.txt
@@ -57,3 +57,4 @@ the "git archive" command.
 Fixed various debian packaging issues (Addresses Debian Bug: #905195)
 
 Update Danish, Spanish, and Swedish translations.
+
